src/scep/Client/client.py

```diff
@@ -176,7 +176,8 @@ def __pki_operation(self, data, cacaps):
             res = requests.post(self.url, params={'operation': 'PKIOperation', 'message': ''}, data=data, headers=headers)
         else:
             b64_bytes = base64.b64encode(data)
-            b64_string = b64_bytes.encode('ascii')
+            b64_string = b64_bytes.decode('utf-8')
```

Comment:
`b64_bytes.decode('ascii')` is probably better

I'm surprised the original code had this encode vs decode bug.

Reply:
I reverted this